src/customFileParser/unpackGeometry/Cone.ts

```diff
@@ -1,37 +1,115 @@
 import PropertyLoader from '../PropertyLoader';
 import * as THREE from 'three';
-import { PrimitiveGroup, ConeGroup, EccentricConeGroup, CircleGroup } from '../../geometry/GeometryGroups';
+import { PrimitiveGroup, ConeGroup, EccentricConeGroup, CircleGroup, GeneralRingGroup, TrapeziumGroup }
+  from '../../geometry/GeometryGroups';
 import { FilterOptions } from '../../parsers/parseUtils';
+import { xAxis, zAxis } from './../../constants';
 
 const centerA = new THREE.Vector3();
 const centerB = new THREE.Vector3();
 
-function addOpenCone(groups: {[name: string]: PrimitiveGroup}, data: PropertyLoader, filterOptions?: FilterOptions) {
+const axisRotation = new THREE.Quaternion();
+const localXAxis = new THREE.Vector3();
+const globalVertex = new THREE.Vector3();
+const rotation = new THREE.Quaternion();
```

Comment:
Let's use the `global` naming convention when we add new things. I know we don't have it everywhere, but it seemed like the best way to avoid shadowing.

Reply:
Fixed everywhere